video/out/vo_gpu_next.c

```diff
@@ -2226,6 +2226,12 @@ static void update_render_options(struct vo *vo)
 #else
     pars->params.blend_against_tiles = opts->background == BACKGROUND_TILES;
 #endif
+    pars->params.tile_size = opts->background_tile_size * 2;
+    for (int i = 0; i < 2; ++i) {
```

Comment:
MP_ARRAY_SIZE?

Reply:
Wouldn't changing layout of any of this arrays possibly break the result anyway? It's fine tho.